JSDemos/Demos/DateRangeBox/Overview/description.md

```diff
@@ -0,0 +1,21 @@
+The [DateRangeBox](/Documentation/ApiReference/UI_Components/dxDateRangeBox/) component allows users to enter or pick dates in a specified range. This demo illustrates the following DateRangeBox properties:
+
+- [value](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#value)     
+An array where you can specify the selected range (start and end dates). The DateRangeBox allows you to define the start and end dates separately. For this purpose, use the [startDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#startDate) and [endDate](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#endDate) properties.
+
+- [displayFormat](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#displayFormat)        
+A date display format. You can use one of the [predefined formats](/Documentation/ApiReference/Common/Object_Structures/format/#type) or specify a [custom format](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String). This demo illustrates the latter.
+
+- [disabled](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#disabled)        
+Specifies whether the DateRangeBox responds to user interaction.
+
+- [showClearButton](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#showClearButton)        
+This property allows you to display the button that clears the DateRangeBox **values**.
```

Comment:
Other descriptions don't start with "this property", so let's continue the same pattern for consistency. 

 ```suggestion
Displays the button that clears DateRangeBox **values**.
```